applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -43,26 +45,67 @@ Syntax
 * The ``<apn>`` parameter is a string.
   It represents an alternative access point name (APN), other than the default primary APN, to use for downloading.
 
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+  #XFOTA: <primary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+  #XFOTA: <secondary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+
+* The ``<primary_partition_id>`` and ``<secondary_partition_id>`` integers that give the partition ID.
+* The ``<MCUBOOT_version>`` integer. The value 1 corresponds to MCUboot versions 1.x.y.
```

Comment:
Is only major version specified? I'm not sure if the entire <Major.Minor.Patch> counts as an integer, float, or neither.

Reply:
Or string.

Reply:
Unfortunate only major version, [link](https://github.com/nrfconnect/sdk-zephyr/blob/master/include/dfu/mcuboot.h#L121)

Reply:
Maybe `The ``<MCUBOOT_version>`` major version integer. Value 1 corresponds to MCUboot versions 1.x.y.

Reply:
Updated as suggested